src/components/admin/ZeroOrder.vue

```diff
@@ -210,6 +210,22 @@
           />
           <span class="checkmark"></span>
         </label>
+      </div>
+      <div>
+        <div class="mr-1">
+          <input
+            class="q-i"
+            type="checkbox"
+            id="return_order"
+            name="return_order"
+            v-bind:checked="return_order"
+            @change="return_order = !return_order"
```

Comment:
why do you need both v-bind and @change ?

Reply:
I copied this functionality from sales->order and it has these lines

Reply:
please correct this. I don't think we need @change when we are using v-bind.
test this also.